### DISCUSSION TITLE
How should we engage with higher education?

### DISCUSSION THREAD

**Discussion by Sealjay** [Green-Software-Foundation/oc#16] [General]
@greenhsu123 :
> I think we have seen a couple of similar asks from UK universities regarding providing speaker, training, material, lab etc… @Tammy/Chris- could we add this as an discussion point in the upcoming OC meeting?

> Personally, I think this is an area we should focus on, especially for the community WG, but we should do it in a more efficient and effective way to avoid any duplicate effort. Maybe some sort of process/template and ready-made material?

**Comment by Sealjay** [comment]
I agree, I think ready-made templates and content are the way forward – we can repurpose some of the GSF summit content, and I can share the deck I used previously too.

**Reply by greenhsu123** [reply]
One of the P & P project's following focuses is on turning our material into a presentation deck and connecting with SCI folks to map Principles & Patterns to an SCI use case. So, a lot more material on our horizon :)

**Reply by Sealjay** [reply]
@adamj89

**Comment by tmcclell** [comment]
@jawache - Question about what we would do for this differently than we already do? Ideas for hackathons? Under 18 prize? @srini1978 - different schools will want different levels of curriculum or engagement. @greenhsu123 Student intern sponsorship. @atg-abhishek - Prize possibly could be an interview opportunity. @Willmish - 48 hour hackathons (locally) @tmcclell - Meetup opportunity. @Henry-WattTime - research on green software pings.

**Reply by Willmish** [reply]
RE: Research on Green Software - as  I am finishing my degree next year, I am looking for a Master Thesis/Project idea and would love to see some more specific on Green Software. This would probably require more coordination with the actual universities as I don't know how research supported by open source communities/foundations functions.

My current topic/project idea is on Rust language's efficiency, but linking it closer to Green Software could be really interesting.

**Comment by Willmish** [comment]
Recently ran a workshop/talk during UCL's Computer Science Society Hackathon on Green Software and Carbon Awareness :
https://www.eventbrite.co.uk/e/ucl-css-x-techsoc-hackathon-engineers-are-superheroes-tickets-529604931607

So definitely we could look more into hosting/supporting hackathons with Universities or University Clubs/Societies

**Reply by akshitbatra** [reply]
Really good idea @Willmish, we can also see communities like [MLH(Major League Hacking)](https://mlh.io/), they organize Global Hack/Local Hack weeks which includes tasks and session for the whole week and have a great community and reaches a lot of students/technologies across the globe.

**Comment by jawache** [comment]
Adding in @russelltrow for visibility on this issue. I think as we explore updating the learn platform and principles content we at least ensure higher education is not excluded. I think this just means when we create the new materials, just making sure to reach out to them and make sure they know we have something for them.

@adamj89 something to remember, something we can easily action given our current pipeline and roadmap is to make sure during the next hackathon that we have prizes specifically for people who are still in education (18s and under prizes). Given that last year 2/3 top prizes were phd I would also argue that we make sure the top prizes don't just go to phds who just submit their phd thesis as a hackathon solution. Not sure we're adding anything to the world there, the work would have happened anyway.
